src/components/ReanimatedSwipeable.tsx

```diff
@@ -262,28 +280,20 @@ const Swipeable = forwardRef<SwipeableMethods, SwipeableProps>(
       },
     });
 
-    const defaultProps = {
-      friction: 1,
-      overshootFriction: 1,
-    };
-
-    const {
-      friction = defaultProps.friction,
-      overshootFriction = defaultProps.overshootFriction,
-    } = props;
-
+    // %% remove or rename these, may cause 1. reloads, 2. name shadowing
```

Comment:
What about this? Also, such comments should be marked as `TODO`

Reply:
Same here

Reply:
Removed this comment in [09f145b](https://github.com/software-mansion/react-native-gesture-handler/pull/3165/commits/09f145b8d0a930d21a219a53d5ab74958d5e7a9c)